### DISCUSSION TITLE
JSON Editor?

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#6804] [Help]
Hi there.

Is there any support for editing JSON? I have a config string that is stored in a json column in my database:

Example:

```json
{"name": "document_workflow", "type": "state_machine"}
```

I have this cast defined on my model:

```php
protected $casts = [
    'config' => 'array'
];
```

**Comment by danharrin** [comment]
One option could be the `KeyValue` field. Or you can use multiple fields like `TextInput::make('config.name')`
